internal/core/entities/game_room/game_room.go

```diff
@@ -25,6 +25,8 @@ package game_room
 import (
 	"fmt"
 	"time"
+
+	portsErrors "github.com/topfreegames/maestro/internal/core/ports/errors"
```

Comment:
We should use those errors only for Ports/Adapters, which is not the case here. If you feel like we need the same kind o package for defining errors on the entities/services, we can set up a different one.